### PR TITLE
fix: consolidate workflows

### PR DESCRIPTION
**Pull request by MichaelGoberling**
This PR consolidates the following workflows: 
- `daily.yml`
- `node.js.yml`
- `run-once.yml` 

Into one workflow file:
- `app-test.yml`

Note: This change attempts to maintain the behavior that only the `daily` and `run-once` workflows notify over Slack and only the `node.js` workflow uploads to Codecov. 

## Related Issue

Closes https://github.com/adobe/aio-e2e-tests/issues/52

## How Has This Been Tested?

This will be the test and... Later will also be the test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/app-test.yml [hunk 1]
```diff
@@ -42,22 +47,25 @@ jobs:
...
         git config user.email github-actions@github.com
         echo "- $(date) Node ${{ matrix.node }} ${{ job.status == 'success' && '🎉 success' ||  job.status }}" >> logs/run.md
         git add logs/run.md
         git commit -m "generated"
         git push
+    - name: Upload Coverage
+      if: ${{ success() && (github.event_name == 'pull_request' || github.event_name == 'push') }}
+      uses: codecov/codecov-action@v3.1.1
+      with:
+        name: ${{ runner.os }} node.js ${{ matrix.node-version }}
```
Comment (shazron) [inline .github/workflows/app-test.yml:63]: `matrix.node-version` should be `matrix.node` (either that or change the var name on line 18)

**Review by shazron — CHANGES_REQUESTED** [review]

File: .github/workflows/app-test.yml (hunk 1 above)
Comment (MichaelGoberling) [reply .github/workflows/app-test.yml:63]: Ah... After all this time looking at it too. Fixing

Comment (MichaelGoberling) [reply .github/workflows/app-test.yml]: Okay fixed

**Comment by MichaelGoberling** [comment]
`aio-lib-runtime` failed with a 404 and 409 from the runtime api... Re-running and crossing my fingers

**Comment by MichaelGoberling** [comment]
Updated the workflow file to only run one version of Node.

Checked and this is consistent with what the original workflow files were doing.

**Review by shazron — APPROVED** [review]

**Pull request merged**